api/include/mfxstructures.h

```diff
@@ -1929,9 +1929,20 @@ typedef struct {
 
 typedef struct {
     mfxExtBuffer Header;
-
     mfxU16 Enable;        /* tri-state option */
-    mfxU16 reserved[11];
+    union {
```

Comment:
So... We DID have an uAPI for enable/disable color fill, but we could not program which color to use (I guess we had some default, right?) and I am not sure whether we use the enable/disable uAPI at all. Considering all that I believe we need to break this change into step-by step:
1. I think you have some code cleanup which precedes your further changes - this goes to separate commit
2. Looks like you have or can provide enable/disable color fill support (without programming the color) within existing uAPI - that's the second commit
3. You have new uAPI added to program color to use in color fill - that's the third commit.